src/table/utils/styling-utils.js

```diff
@@ -122,19 +149,31 @@ export function getColumnStyle(styling, qAttrExps, stylingInfo) {
   return {
     ...styling,
     color: columnColors.cellForegroundColor || styling.color,
-    background: columnColors.cellBackgroundColor,
+    backgroundColor: columnColors.cellBackgroundColor,
   };
 }
 
-export function getSelectionColors(cell, selectionState, background, tableBackgroundColor) {
+/**
+ * Get different styles for cells based on wether they are
+ * selected or possible to be selected or not possible to be selected
+ * @param {Object} cell - each cell in the table body
+ * @param {Object} selectionState - the selected cell
+ * @param {?String} columnBackgroundColor - the background color from qAttributeExpressions in qDimensions or qMeasures
+ * @param {?String} themeBackgroundColor - the background color from nebula theme or sense theme
+ * @returns {Object} different set of styles for different state of cells
```

Comment:
Also a bit confusing. maybe just `styling for the provided cell`

Reply:
yup